src/main/java/teammates/logic/backdoor/BackDoorServlet.java

```diff
@@ -133,149 +52,145 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         resp.flushBuffer();
     }
 
-    private String executeBackendAction(HttpServletRequest req, String action)
-            throws EntityDoesNotExistException, InvalidParametersException, IOException {
-        // TODO: reorder in alphabetical order
+    @SuppressWarnings("PMD.SwitchStmtsShouldHaveDefault") // no default so that each case is accounted for
```

Comment:
It is. I believe the rule is tailored for non-enumerable types (Strings, integers) in which no default would indeed be disastrous.